pkg/antctl/raw/proxy/command.go

```diff
@@ -132,6 +132,12 @@ func init() {
 		Long:    "Run a reverse proxy to access Antrea API (Controller or Agent). Command only supports remote mode. HTTPS connections between the proxy and the Antrea API will not be secure (no certificate verification).",
 		Example: proxyCommandExample,
 		RunE:    runE,
+		Args: func(cmd *cobra.Command, args []string) error {
```

Comment:
can't we use the builtin `args: NoArgs` command validator here? See https://pkg.go.dev/github.com/spf13/cobra#readme-positional-and-custom-arguments

Reply:
Yes. Didn't notice that before.